pkg/informers/informers.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2020 The prometheus-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package informers
+
+import (
+	"strings"
+	"time"
+
+	"github.com/prometheus-operator/prometheus-operator/pkg/listwatch"
+
+	"github.com/pkg/errors"
```

Comment:
nit, we can cuddle this with the packages below, no?

Reply:
fixed